backend/nestjs/pong/src/app/projects/game/game.gateway.ts

```diff
@@ -54,19 +63,13 @@ export class GameGateway implements OnGatewayConnection, OnGatewayDisconnect {
 			this.server.emit(uuid.toString(), response.gameState.id);
 		}
 		catch (error) {
-			this.server.emit(uuid.toString(), JSON.stringify(
-				{
-					"COULDN'T JOIN THE GAME": [error.message],
-				}
-			));
+			console.log(`COULDN'T JOIN THE GAME: ${[error.message]}`);
```

Comment:
SUGESTÃO: Existe o pacote Logger no @nestjs/common, vc consegue passar pro construtor dele o nome da instancia que esta executando o log e consegue personalizar o tipo do log como log, warn e erro por exemplo

Reply:
To ligado mano, coloquei ai só porque foi mais rápido, se der tempo eu altero.